models/db/log.go

```diff
@@ -14,15 +15,19 @@ import (
 
 // XORMLogBridge a logger bridge from Logger to xorm
 type XORMLogBridge struct {
-	showSQL bool
-	logger  log.Logger
+	showSQLint *int32
```

Comment:
I don't really know what the benefit of converting this to an int pointer is.
Wouldn't a bool pointer be easier to use?
Or am I missing something?

Reply:
sync/atomic does not have bool pointer based functions. 

int32 pointers are well supported by atomic instead.